code/datums/riding.dm

```diff
@@ -252,6 +252,71 @@
 					buckled_mob.pixel_x = 10
 					buckled_mob.pixel_y = 5
 
+//prototype repair vehicle
+/datum/riding/space/repair
+	keytype = null
+	vehicle_move_delay = 1
+	var/obj/machinery/repair_turret/turret = null
+	var/obj/vehicle/space/speedbike/repair/R = null
+
+/datum/riding/space/repair/handle_vehicle_layer()
+	R = ridden
+	turret = R.turret
```

Comment:
The best I can do is change all references from this turret var to R.turret, which I don't know what the point of that would be. 

Unless you can show me what a working alternative could be. I literally spent hours re-arranging these two vars in the datum to get them to work, they're currently bugged on live but since turret ATV's are admin-spawn nobody has noticed.